packages/eds-core-react/src/components/next/Input/Input.types.ts

```diff
@@ -0,0 +1,34 @@
+import { ReactNode, CSSProperties, ComponentPropsWithoutRef } from 'react'
+
+type ElementComponent = 'input' | 'textarea'
+
+export type InputProps = {
```

Comment:
I recommend having a look at the existing Input component where we extend InputHTMLAttributes. Then you don't have to specify these properties.